dask/bag/core.py

```diff
@@ -1101,26 +1101,32 @@ def from_castra(x, columns=None, index=False, npartitions=None):
         x = Castra(x.path, readonly=True)
     if columns is None:
         columns = x.columns
-    if npartitions is None:
```

Comment:
Castra is generally low-ish priority from my perspective.  I would go with the cheaper option.